ssl/statem/extensions.c

```diff
@@ -357,11 +357,12 @@ static int extension_is_relevant(SSL *s, unsigned int extctx,
  * found, or an internal error occurred. We only check duplicates for
  * extensions that we know about. We ignore others.
  */
-int tls_collect_extensions(SSL *s, PACKET *packet, unsigned int context,
+size_t tls_collect_extensions(SSL *s, PACKET *packet, unsigned int context,
```

Comment:
This has theh type match with clienthello.pre_proc_exts_len, which is what we're going to assign it to, but I agree that int is going to be big enough to hold the values that would be seen.